sources/dynamodb/schema.go

```diff
@@ -188,6 +191,14 @@ func (isi InfoSchemaImpl) ProcessData(conv *internal.Conv, srcTable string, srcS
 	return nil
 }
 
+func (isi InfoSchemaImpl) StartChangeDataCapture(ctx context.Context, conv *internal.Conv) (map[string]interface{}, error) {
```

Comment:
We will need to modify the source profile conn for dynamodb in case of streaming will that be a part of different PR ?

Reply:
As we are planning to introduce a `streaming` flag for starting streaming migration. We can parse this flag in the main cmd package and change the existing SourceProfile.Conn.Streaming flag to start streaming migration for Dynamodb